etcdctl/go.mod

```
@@ -1,8 +1,6 @@
 module go.etcd.io/etcd/etcdctl/v3
 
-go 1.24
-
-toolchain go1.24.6
+go 1.25
```

Please add the `toolchain` directive to this file.

```suggestion
go 1.25

toolchain go1.25.0
```